icepyx/core/is2ref.py

```diff
@@ -77,30 +77,33 @@ def _get_custom_options(session, dataset, version):
     format_vals = [formats[i]["value"] for i in range(len(formats))]
     format_vals.remove("")
     cust_options.update({"fileformats": format_vals})
-
-    # reprojection only applicable on ICESat-2 L3B products, yet to be available.
-
-    # reformatting options that support reprojection
-    normalproj = [Projections.attrib for Projections in root.iter("Projections")]
-    normalproj_vals = []
-    normalproj_vals.append(normalproj[0]["normalProj"])
-    format_proj = normalproj_vals[0].split(",")
-    format_proj.remove("")
-    format_proj.append("No reformatting")
-    cust_options.update({"formatreproj": format_proj})
+    
+    # reprojection only applicable on ICESat-2 L3B products.
 
     # reprojection options
-    projections = [Projection.attrib for Projection in root.iter("Projection")]
+    projections = [Projection.attrib for Projection in root.iter('Projection')]
```

Comment:
formatting inconsistencies that are fixed by black during the pre-commit run to make it clearer where code has been changed in the diff view (i.e. what we're looking at)